Plugins/Creature/Creature.cpp

```diff
@@ -1902,5 +1907,79 @@ ArgumentStack Creature::GetFlatFooted(ArgumentStack&& args)
     return Services::Events::Arguments(retVal);
 }
 
+ArgumentStack Creature::SerializeQuickbar(ArgumentStack&& args)
+{
+    std::string retVal;
+
+    if (auto *pCreature = creature(args))
+    {
+        uint8_t *pData = nullptr;
+        int32_t dataLength = 0;
+
+        CResGFF    resGff;
+        CResStruct resStruct{};
+
+        if (resGff.CreateGFFFile(&resStruct, "GFF ", "V2.0"))
+        {
+            pCreature->SaveQuickButtons(&resGff, &resStruct);
+            resGff.WriteGFFToPointer((void**)&pData, /*ref*/dataLength);
```

Comment:
Yep, this leaks `pData`. Needs a `delete [] pData;` after encoding. Same problem in Serialize.cpp

Reply:
Fixed